plugins/convention/src/main/kotlin/com/bumble/appyx/multiplatform/MultiplatformConventionPlugin.kt

```diff
@@ -0,0 +1,29 @@
+package com.bumble.appyx.multiplatform
+
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+
+class MultiplatformConventionPlugin : Plugin<Project> {
+    override fun apply(project: Project) {
+        project.extensions.create("appyx", MultiplatformConventionExtension::class.java)
+
+        project.plugins.apply("kotlin-multiplatform")
+        project.plugins.apply("org.jetbrains.compose")
+        project.plugins.apply("appyx-detekt")
+
+        project.extensions.configure<DetektExtension> {
```

Comment:
If we create an iOS example in the future this list would need to be updated. annoyingly detekt doesn't look at all directories

Reply:
I wonder what would be the best place to record this memo, here it will just get lost.

Reply:
Good question. I'm unsure. Do we have a milestone or something for any planned iOS tickets? It could be added there?